src/ethereum/ethEngine.js

```diff
@@ -622,9 +622,9 @@ export class EthereumEngine extends CurrencyEngine {
     )
     const wallet = ethWallet.fromPrivateKey(privKey)
 
-    this.log(wallet.getAddressString())
+    this.log.warn(wallet.getAddressString())
```

Comment:
Add some context to this log. ie 

    this.log.warn(`signTx getAddressString ${wallet.getAddressString()}`)